internal/pipeline/executor.go

```diff
@@ -138,9 +140,19 @@ func (p *pkg) resolveSources() []string {
 			// include only this pkg sources
 			pkgPaths = append(pkgPaths, p.Path())
 		case sourceAllSubPkgs:
-			// TODO(droot): figure out how to enumerate sub packages for current pkgPath.
-			// For now including current pkg
+			// For now including current pkg and subpkgs in current directory
 			pkgPaths = append(pkgPaths, p.Path())
+			files, _ := ioutil.ReadDir(p.Path())
+			for _, f := range files {
+				if f.IsDir() {
+					// A directory is a package if it has a Kptfile.
+					// This may change as the concept of a package is expanded such that
+					// every directory is its own kpt package
+					subPaths, _ := pathutil.DirsWithFile(p.Path()+"/"+f.Name(), "Kptfile", false)
```

Comment:
Use the `path` pkg helper functions to construct these paths (I think there is `Join`) method, it will help us later to support different types of OS etc. better.

Reply:
Also, don't ignore error :)

Reply:
done, thanks!